src/core/ucc_lib.c

```diff
@@ -353,6 +353,14 @@ ucc_status_t ucc_init_version(unsigned api_major_version,
         goto error;
     }
 
+    status = ucc_mpool_init(&lib->stub_tasks_mp, 0, sizeof(ucc_coll_task_t), 0,
+                            UCC_CACHE_LINE_SIZE, 8, UINT_MAX,
```

Comment:
do we need 8 elems_per_chunk for zero size messages?

Reply:
8 is enough and it doesn't consume much memory